backend/pkg/utils/helpers.go

```diff
@@ -49,3 +53,63 @@ func CallPredictionAgentService(jsonData []byte) (analysisResponse models.IssueA
 	}
 	return
 }
+
+func CompareLogs(incomingLogTails []string, currentIssuesLogTails []string) (isNewIssue bool) {
+	logSimilarityThreshold := 0.8
+	isNewIssue = true
+	sdm := metrics.NewSorensenDice()
+	sdm.CaseSensitive = false
+	sdm.NgramSize = 3
+	for _, incomingLogTail := range incomingLogTails {
+		for _, currentIssueLogTail := range currentIssuesLogTails {
+			similarity := strutil.Similarity(incomingLogTail, currentIssueLogTail, sdm)
+			if similarity >= logSimilarityThreshold {
+				isNewIssue = false
+			}
+			fmt.Print("---------------------\n")
+			fmt.Printf("analyzedLog: %s \nissueLog: %s \n score: %.2f \nisNewIssue: %v", incomingLogTail, currentIssueLogTail, similarity, isNewIssue)
+			fmt.Print("---------------------\n")
+			if similarity >= logSimilarityThreshold {
```

Comment:
Is that if statement redundant? We have one few lines above

Reply:
Ah yes the first one was for testing purposes didn't remove it